scripts/bump.sh

```diff
@@ -49,16 +49,16 @@ jq ".version = \"$NEXT_EXTENSION_VERSION\" | \
     ./server/package.json > ./server/package.json.bk
 
 jq ".version = \"$NEXT_EXTENSION_VERSION\"" \
-    ./client/package.json > ./client/package.json.bk
+    ./clients/vscode/package.json > ./clients/vscode/package.json.bk
 
 mv ./package.json.bk ./package.json
 mv ./server/package.json.bk ./server/package.json
-mv ./client/package.json.bk ./client/package.json
+mv ./clients/vscode/package.json.bk ./clients/vscode/package.json
```

Comment:
If I understand correctly, the only structural change in this PR is that clients get a further sub directory for `vscode` extension. This code should be enough for release publish to update the new folder. 

Is there any other directory structure change?

Reply:
Noted another thing. We no longer want to edit the `package.json` file at the root right? as we removed this information (name, displayName, version etc) from there?

Reply:
> If I understand correctly, the only structural change in this PR is that clients get a further sub directory for `vscode` extension. This code should be enough for release publish to update the new folder.
> 
> Is there any other directory structure change?

There is another change. Every time the language server gets published, the vscode extension should get published as well. So we might need a script publishing both the extension and the language server.